test/polygon/sandPolyonDepositor/fixtures.ts

```diff
@@ -0,0 +1,70 @@
+import {
+  ethers,
+  deployments,
+  getUnnamedAccounts,
+  getNamedAccounts,
+} from 'hardhat';
+import {Contract, BigNumber} from 'ethers';
+import {transferSand} from '../../catalyst/utils';
+
+export const setupSandPolygonDepositor = deployments.createFixture(async () => {
+  await deployments.fixture();
+  const sandContract: Contract = await ethers.getContract('Sand');
+  const users = await getUnnamedAccounts();
+  const {catalystMinter} = await getNamedAccounts();
+  const user0 = users[0];
+
+  await deployments.deploy(`MockSandPredicate`, {
+    contract: 'MockSandPredicate',
+    from: catalystMinter,
+    log: true,
+    args: [],
+  });
+
+  const mockSandPredicateContract: Contract = await ethers.getContract(
+    'MockSandPredicate'
+  );
+
+  await deployments.deploy(`MockRootChainManager`, {
+    contract: 'MockRootChainManager',
+    from: catalystMinter,
+    log: true,
+    args: [mockSandPredicateContract.address],
+  });
+
+  const mockRootChainManagerContract: Contract = await ethers.getContract(
+    'MockRootChainManager'
+  );
+
+  await deployments.deploy(`SandPolygonDepositor`, {
```

Comment:
What about a deploy script in the deploy folder for `SandPolygonDepositor.sol`?

Reply:
I will be adding this in the future, there is a dependency on the `SandPredicate` contract.